moped-database/migrations/1680199250609_add_new_subphases/down.sql

```diff
@@ -0,0 +1,3 @@
+DELETE FROM moped_subphases WHERE subphase_name = 'Requested by partner';
+DELETE FROM moped_subphases WHERE subphase_name = 'Fiber/Communications';
+DELETE FROM moped_subphases WHERE subphase_name = 'Detection/CCTV';
```

Comment:
These might need an additional condition for their `related_phase_id` since subphase name isn't unique from row to row. Like how `Study in progress` is the value of two rows in this table.

Probably never going to run this down migrations but just in case.

Reply:
good catch @mddilley. agreed we will never run this migration but it's probably best to have it right.